v03_pipeline/lib/tasks/base/base_loading_run_params.py

```diff
@@ -34,6 +34,10 @@ class BaseLoadingRunParams(luigi.Task):
         default=False,
         parsing=luigi.BoolParameter.EXPLICIT_PARSING,
     )
+    skip_expect_tdr_metrics = luigi.BoolParameter(
+        default=False,
+        parsing=luigi.BoolParameter.EXPLICIT_PARSING,
+    )
```

Comment:
what's the logic here? will it be possible to not use TDR metrics and pass in a TSV directly ?

Reply:
yes, we actually can pass in a tsv directly.  The tsvs are stored, one per TDR "dataset" here: https://github.com/broadinstitute/seqr-loading-pipelines/pull/999/files#diff-583758b4dea21ca2ae6729f0255825672e2de91bdc96b90aab88a8657f2f3cf5R115

Reply:
We'll be able to modify them if need be, though we really shouldn't.